<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Matus &amp; Michal Tomlein</author>
 <class>AboutWidget</class>
 <widget class="QWidget" name="AboutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Leaklog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/resources.qrc">
    <normaloff>:/images/images/help_about.png</normaloff>:/images/images/help_about.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#AboutWidget { background-color: white; }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="margin">
      <number>12</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_icon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rc/resources.qrc">:/images/images/leaklog256.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="margin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_szchkt_icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="../rc/resources.qrc">:/images/images/szchkt.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_leaklog">
       <property name="font">
        <font>
         <pointsize>34</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #414141;</string>
       </property>
       <property name="text">
        <string>Leaklog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_version">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #9F9F9F;</string>
       </property>
       <property name="text">
        <string>Version %1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QWebEngineView" name="webv_about">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">.QWidget { background-color: #EDEDED; border-color: #BFBFBF; border-style: solid; border-width: 1px 0px 0px 0px; }</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="margin">
       <number>12</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_about_qt">
        <property name="text">
         <string>About Qt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_acknowledgements">
        <property name="text">
         <string>Acknowledgements</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_licence">
        <property name="text">
         <string>Licence</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
